pkg/qemu/qemu.go

```diff
@@ -1119,7 +1119,7 @@ func getFirmware(qemuExe string, arch limayaml.Arch) (string, error) {
 		// Fedora package "edk2-ovmf"
 		candidates = append(candidates, "/usr/share/edk2/ovmf/OVMF_CODE.fd")
 		// openSUSE package "qemu-ovmf-x86_64"
-		candidates = append(candidates, "/usr/share/qemu/ovmf-x86_64-code.bin")
+		candidates = append(candidates, "/usr/share/qemu/ovmf-x86_64-4m.bin")
```

Comment:
What was the actual problem with code.bin?

Reply:
For consistency with other distros, can we just keep code and add vars ?

Reply:
> For consistency with other distros, can we just keep code and add vars ?

You want to add `vars` for all the distros?

Reply:
Was this a change in qemu, or where does the qemu-ovf upstream live? Looks like an old issue/bug was fixed...

Reply:
This seems like a lot of complexity. Why not use the unified firmaware on all platforms instead? What is the advantage of splitting it?

Reply:
> Was this a change in qemu, or where does the qemu-ovf upstream live? Looks like an old issue/bug was fixed...

I believe the reason `ovmf.bin` can be split into `ovmf-code.bin` and `ovmf-vars.bin` is so that you can actually persist NV parameters for a virtual machine. It is supposed to be used like this

```console
$ cp /usr/share/qemu/ovmf-x86_64-vars.bin .
$ qemu-system-x86_64 \
    -drive if=pflash,format=raw,unit=0,readonly,file=/usr/share/qemu/ovmf-x86_64-code.bin \
    -drive if=pflash,format=raw,unit=1,file=ovmf-x86_64-vars.bin
```

For some reason there was a patch the did automatically reserve space for NV variables in the `ovmf-code.bin` software, so you could use the file by itself (of course changes to NV variables will not persist in this case). I have no idea how this patched CODE image would be different from the combined/unified image.

Now it seems like this patch is interfering with secure boot in some configurations, so has been removed from the openSUSE builds, and the CODE version no longer works unless you also mount the VARS image as well.

It is not clear to me why we don't use the unified image everywhere (beyond that we have to figure out if it is available on the distro, and what names are being used).